server/rest/src/main/java/org/infinispan/rest/resources/MetricsResource.java

```diff
@@ -71,19 +48,12 @@ public Invocations getInvocations() {
    }
 
    private CompletionStage<RestResponse> metrics(RestRequest restRequest) {
-      List<String> accept = responseMediaTypes(restRequest);
-
-      CompletableFuture<RestResponse> cf = CompletableFuture.supplyAsync(() -> {
+      return CompletableFuture.supplyAsync(() -> {
          RestResponseBuilder<NettyRestResponse.Builder> builder = new NettyRestResponse.Builder();
 
          try {
-            requestHandler.handleRequest(restRequest.path(), restRequest.method().name(),
-                                         accept.stream(), (status, message, headers) -> {
-                     builder.status(status).entity(message);
-                     for (String header : headers.keySet()) {
-                        builder.header(header, headers.get(header));
-                     }
-                  });
+            builder.header("Content-Type", TextFormat.CONTENT_TYPE_004);
```

Comment:
I am not familiar with this content type, can we get a comment? From the entity I was thinking TEXT_PLAIN, but I haven't looked closely.

Reply:
CONTENT_TYPE_004 is the Prometheus content type. Sure, let's add a comment here!